### PR TITLE
Some minor mustachio cleanup

### PR DESCRIPTION
**Pull request by parlough**
Some minor, non-consequential cleanup to some mustachio code.

@srawlins

### CODE REVIEW

File: lib/src/mustachio/parser.dart [hunk 1]
```diff
@@ -177,7 +177,7 @@ class MustachioParser {
   ///
   /// [_index] should be at the character immediately following the `#`
   /// character which opens a possible section tag.
-  _TagParseResult _parseSection({@required invert}) {
+  _TagParseResult _parseSection({@required bool invert}) {
```
Comment (srawlins) [inline lib/src/mustachio/parser.dart:180]: *cringe* yikes, thanks!

**Review by srawlins — APPROVED** [review]
Love it, thanks!

File: lib/src/mustachio/parser.dart (hunk 1 above)
Comment (parlough) [reply lib/src/mustachio/parser.dart:180]: @pq Is this something that `type_annotate_public_apis` should have caught?

Comment (jcollins-g) [reply lib/src/mustachio/parser.dart:180]: you'd think there should be a "no implicit dynamic" lint of some sort.   At least, whenever I write an implicit dynamic it's almost always an accident.

**Pull request merged**